src/ext/java/org/opentripplanner/ext/transmodelapi/TransmodelGraphQLSchema.java

```diff
@@ -1299,7 +1298,8 @@ private GraphQLSchema create() {
                 lineIds == null || lineIds.isEmpty() || lineIds.contains(t.getRoute().getId())
               )
               .filter(t ->
-                CollectionUtils.isEmpty(privateCodes) ||
```

Comment:
This could probably move into our own CollectionUtils class.

Reply:
I made sure, that these arguments are always set to an empty list, if they are null, when fetching the argument from the environment.